notebooks/growth_model_classification.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<p>shouldn't we be calculating the slope just for the 6month prediction window (i.e. <code>2015-04 to 2015-09</code>) and exclude the validation months i.e remove <code>2015-01 to 2015-03</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Good catch. Thanks Hema for pointing this out. I have now changed it.